src/gui/menu_color.cpp

```diff
@@ -31,13 +33,32 @@ ColorMenu::ColorMenu(Color* color_) :
   add_color_channel_rgba(&(color->alpha), Color::BLACK, -1, true);
   add_color_display(color);
 
+  add_hl();
+  add_item(std::make_unique<MenuItem>(_("Copy"), 1));
+  add_item(std::make_unique<MenuItem>(
```

Comment:
Use `add_entry` here for consistency (the color argument from the constructor should be available in `Menu::add_entry` too). Also maybe MNID_ enumerators like other menus?